File: programming files/sum_even_fibonacci.java
public class sum_even_fibonacci {
    public static void main(String[] args) {
        int a = 0, b = 1, c = 2;
        int sumEven = 0;
        while (c < 4000000) {
            if (c % 2 == 0) {
                sumEven += c;
            }
            int temp2 = c;
            int temp1 = b;
            c = a + b + c;
            a = temp1;
            b = temp2;
        }
        System.out.println(sumEven);
    }
}

// answer should be 4613732

File: programming files/sum_even_fibonacci.py
//
a, b = 0, 1
//starting the sum at 0
sum_even = 0
//while loop
while b < 4000000:
    if b % 2 == 0:
        sum_even += b
    a, b = b, a+b

print(sum_even)
#predicted answer
# answer should be 4613732
